postcss/es-transforms-rems.js

```diff
@@ -0,0 +1,9 @@
+const postcss = require('postcss');
+
+module.exports = postcss.plugin('es-transform-rems', () => {
+	return (css) => {
+		css.walkDecls((decl) => {
+			decl.value = decl.value.replace(/([0-9.-]+rem)/g, 'calc($1 * var(--base-font-size, 1))');
```

Comment:
I have a feeling that this can't be implemented progressively, just all-or-nothing: what happens if someone has left a `calc(var(--base-font-size) * 3rem);` value in their code and has turned on `useRemBaseSize` in the global manifest?